openebench/APAeval_schemas/aggregation_events/aggregationevent_Q2.json

```diff
@@ -0,0 +1,29 @@
+{
+    "_id": "APAeval:2021-06-04_aggregationEvent_AA_Q2",
+    "_schema": "https://www.elixir-europe.org/excelerate/WP2/json-schemas/1.0/TestAction",
+    "action_type": "AggregationEvent",
+    "challenge_id": "APAeval:2021-06-04",
+    "dates": {
+        "creation": "2021-06-04T00:00:00Z",
+        "reception": "2021-06-04T00:00:00Z"
+    },
+    "tool_id": "APAeval:aggregation",
+    "involved_datasets": [
+        {
+            "dataset_id": "APAeval:2021-06-04_AA_QAPA_Q2_01_corr",
```

Comment:
This should be replaced by the actual `dataset_id`.

Reply:
Same for the other datasets in this file.

Reply:
have a look at any of these examples to check what are the fields that are mandatory:
https://github.com/inab/benchmarking-data-model/tree/master/prototype-data/1.0.x/QfO/Challenges
dataset_id is not necessary I am afraid

Reply:
Not in the challenge schema, in the testAction schema it is. Will leave this for now, as apparently we won't need the testAction schemas